### PR TITLE
Use env file

### PR DESCRIPTION
**Pull request by luckysori**
See comit-network/comit-js-sdk#6 and comit-network/create-comit-app#10 and modify accordingly.

---

Current state:
1. Run `create-comit-app start-env` in root directory (writes `.env` file). Wait until  `Two cnds up and running` is logged.
2. Run `yarn run start`.
3. Fails with `UnhandledPromiseRejectionWarning: Error: invalid extended key (argument="extendedKey", value="[REDACTED]", version=4.0.37)`

Notes: 
- Wait for #19 to be merged.
- Recommend using [`yarn link`](https://yarnpkg.com/lang/en/docs/cli/link/) to work with a local version of comit-js-sdk, as suggested by @thomaseizinger. Before merging remember to depend on a new npm release of comit-sdk.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -47,22 +52,21 @@ import { CoinType, HelloSwap } from "./helloSwap";
...
 
     const ethereumWallet = new EthereumWallet(
-        process.env[`ETHEREUM_HD_KEY_${index}`]!,
+        process.env[`ETHEREUM_KEY_${index}`]!,
         process.env.ETHEREUM_NODE_HTTP_URL!
     );
 
     const app = new HelloSwap(
-        process.env[`HTTP_PORT_CND_${index}`]!,
+        `http://localhost:${process.env[`HTTP_PORT_CND_${index}`]!}`,
```
Comment (luckysori) [inline src/index.ts]: :question: What would we lose by using a `HTTP_URL_CND_*` instead of this?

Comment (D4nte) [reply src/index.ts]: This actually needs to be correct, the `.env` now provides the full url.

Comment (thomaseizinger) [reply src/index.ts]: Done.

**Review by thomaseizinger — COMMENTED** [review]
Good to go in from my side but I wrote all the code.

cc @bonomat @D4nte @luckysori

**Review by luckysori — COMMENTED** [review]
Can't review because I created the original PR, but I approve!

**Pull request merged**